### ISSUE TITLE
Allow weight for Loss 

### ISSUE THREAD

**Issue by M-Colley** [jdb78/pytorch-forecasting#1573]
Internally, for example [CrossEntropy](https://github.com/jdb78/pytorch-forecasting/blob/master/pytorch_forecasting/metrics/point.py) uses _F.cross_entropy_. This allows to **weight** the loss (see also [here](https://github.com/pytorch/pytorch/blob/main/torch/nn/modules/loss.py)). 

A feature request would be to expose this also in pytorch-forecasting as this is quiet often needed IMO.

Kind regards

**Comment by herry73** [comment]
Hey, I would like to try to help to implement this feature. Looks like no one is working on this. Should be simple 
Warm regards.
